docs/coordinates/spectralcoord.rst

```diff
@@ -263,8 +268,8 @@ have been measured (for the purposes of the example here we will assume we have
 11 frequencies)::
 
     >>> sc_ttau = SpectralCoord(np.linspace(200, 300, 11) * u.GHz,
-    ...                         observer=alma, target=ttau)  # doctest: +IGNORE_WARNINGS +REMOTE_DATA
-    >>> sc_ttau  # doctest: +REMOTE_DATA +FLOAT_CMP
+    ...                         observer=alma, target=ttau)  # doctest: +IGNORE_WARNINGS
+    >>> sc_ttau  # doctest: +FLOAT_CMP +REMOTE_DATA
```

Comment:
Like the other example above, `+REMOTE_DATA` here is due to IERS leap seconds table, not `of_site` anymore.

Reply:
I wonder if we cannot set tests to execute with an `offline` or `auto_download=False` config - really, the bundled `IERS` is good enough for these tests so it should not download anything. But obviously `Time` cannot know this...  Anyway, separate issue!